lib/client_side_validations/simple_form/form_builder.rb

```diff
@@ -34,6 +36,14 @@ def wrapper_error_component
           wrapper.find(:full_error)
         end
       end
+
+      def add_field_specific_wrapper_name_to_field_options(attribute_name, options, &block)
+        wrapper_name = options[:wrapper] || find_wrapper_mapping(find_input(attribute_name, options, &block).input_type)
+        return if wrapper_name.nil?
+
+        options[:input_html] ||= {}
+        options[:input_html][:'data-simple-form-wrapper'] = wrapper_name
```

Comment:
Amazing, thanks.

TL;DR: `[:'data-client-side-validations-wrapper']`, move logic to the main gem, use `super` here 

I think that this should be moved to the main gem, because it could be useful even without Simple Form. Then the simple form plugin could only edit options `find_wrapper_mapping(find_input(attribute_name, options, &block).input_type)` and call `super` 

 I would like to keep using the not-so-short `data-client-side-validations` attribute name (IMHO `data-simple-form` gives the idea that SimpleForm added the data attribute, violating the principle of least surprise) and `wrapper` is ok

Feel free to keep doing your experiments here, then you can move to the main gem or I can do it

Reply:
Great, thanks for feedback. I agree with `:'data-client-side-validations-wrapper'` and updated it. Feel free to move this to the main gem. I continue by making radios, checkboxes etc. to work on new branch.